java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/DateConsumer.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Calendar;
 import java.util.Date;
```

Comment:
should probably use `java.sql.Date` instead.

Reply:
good point. thank you.